src/pages/WorkspaceDetails/Header/index.tsx

```diff
@@ -15,7 +15,7 @@ import { PageSection, Text, TextContent } from '@patternfly/react-core';
 import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
 import { ROUTE } from '../../../route.enum';
 import { SECTION_THEME } from '../index';
-import WorkspaceStatusLabel from './WorkspaceStatusLabel';
+import WorkspaceStatusLabel from '../../../components/WorkspaceStatusLabel/WorkspaceStatusLabel';
```

Comment:
`WorkspaceStatusLabel/WorkspaceStatusLabel.tsx` should be renamed as `WorkspaceStatusLabel/index.tsx`, so this import then should be 

```typescript
import WorkspaceStatusLabel from '../../../components/WorkspaceStatusLabel';
```

Reply:
I have fixed it.